Author: Sandro Tosi <morph@debian.org>
Description: Fix manpage as reported in BTS #450532 and #455727 and minus signs
Forwarded: http://code.google.com/p/mp3check/source/detail?r=5
--- mp3check.orig/mp3check.1
+++ mp3check/mp3check.1
@@ -19,7 +19,7 @@
 mp3check \- check mp3 files for consistency
 .SH SYNOPSIS
 .B mp3check
-[\-03ABCEFGIKLMNPRSTWYZabcdefghlmopqrst]  [\-\-accept=LIST] [\-\-alt-color] [\-\-anomaly-check] [\-\-anomaly-check]
+[\-03ABCEFGIKLMNPRSTWYZabcdefghlmopqrst]  [\-\-accept=LIST] [\-\-alt-color] [\-\-anomaly-check]
 [\-\-any-bitrate] [\-\-any\-crc] [\-\-any\-emphasis] [\-\-any-layer] [\-\-any-mode] 
 [\-\-any-sampling] [\-\-any\-version] [\-\-ascii\-only] [\-\-color] [\-\-compact-list] [\-\-cut-junk-end] 
 [\-\-cut-junk-start] [\-\-cut-tag-end] [\-\-dummy] [\-\-dump\-tag] [\-\-dump-header] [\-\-dump-tag] [\-\-edit\-frame\-byte=P]
@@ -70,7 +70,7 @@
 check crc and headers for consistency and print several error messages
 .TP
 .B \-m \-\-max-errors=<int>  
-with -e: set maximum number of errors to print per file (0==infinity) 
+with \-e: set maximum number of errors to print per file (0==infinity)
 (range=[0..])
 .TP
 .B \-a \-\-anomaly-check     
@@ -107,17 +107,18 @@
 .TP
 .B \-\-fix-headers       
 fix invalid headers (prevent constant parameter switching), 
-implies -e, use with care
+implies \-e, use with care
 .TP
 .B \-\-fix-crc           
-fix crc (set crc to the calculated one), implies -e, use with 
+fix crc (set crc to the calculated one), implies \-e, use with
 care (note: it is not possible to add crc to files which have 
 been created without crc)
 .TP
-\fBdisable error messages for -e --error-check:\fP
+\fBdisable error messages for \-e \-\-error\-check:\fP
 .TP
 .B \-G \-\-ign-tag128        
 ignore 128 byte TAG after last frame
+.TP
 .B \-R \-\-ign-resync        
 ignore invalid frame header
 .TP
@@ -145,7 +146,7 @@
 .B \-\-show\-valid
 print the message 'valid audio mpeg stream' for all files which error free (after ignoring errors)
 .TP
-\fBdisable anomaly messages for -a --anomaly-check:\fP
+\fBdisable anomaly messages for \-a \-\-anomaly\-check:\fP
 .TP
 .B \-C \-\-any-crc           
 ignore crc anomalies
